docs/meetings/client/2022_09_23.md

```diff
@@ -0,0 +1,103 @@
+    - The goal of our project will be to **add animation to the avatar**
+        - animated avatar or animation added to still photo with lips syncing to words
+        - User is presented with an embedded video to watch and listen to the resulting animated image with a voice over
+    - The user could potentially upload an image for the avatar they want, but Nazim said our prototype could be limited to default avatars that the user can select from the currently existing web app.
+    - Nazim requested that we research different approaches (animation vs. still image, 2D vs. 3D) as well as how to ensure the timings for lip syncing is appropriate
+
+<br>
+
+4. **Define milestones, deliverables, and success criteria**
+
+    - Nazim provided our professor with the expected milestones, so we need to follow up about it.
+    - Nazim is expecting at least 1 small feature released every 2 weeks
+    - There was a misunderstanding of the length of the Capstone Project; Nazim was expecting a prototype of this project completed by Christmas, but it runs until April 2023
+    - Success criteria:
+        - quick processing
+        - realistic lip syncing for different languages
+        - compares to competitive software (e.g., www.d-id.com/) or similar programs (https://www.youtube.com/watch?v=y3B8YqeLCpY)
+
+<br>
+
+5. **Requirements gathering**
+
+    *Jasper joined the meeting via Nazim's computer and helped provide information*
+
+    - Since they already have the text-to-speech service, our focus is on the animating the image so that the current product is extended to include an animated video clipe with a face moving
+    - Jasper said, "Existing speech API may be usable, but the main interest and focus is how can we create something that gives the user the ability to upload 1-2 images and make it come alive and sync with some audio"
+    - Nazim pointed out that it would be easier to work with default avatars (from where the user creates the actor) so we could let the user select the avatar
+    
+    - After the user submits the avatar, language, voice, and text, the user is presented with a downloadable video with audio
+
+    - The focus is on the back-end application with small front-end to showcase because we will need to create the animation with timings
+    
+    - Programming language expectations:
+        - Back-end is where it would run in the end (currently Node.js with TypeScript, AWS Elastic Beanstalk, etc.)
+        - However, as long as the student project program works with requests, there are no restrictions in terms of languages
+        - If using Typescript, Jasper recommends to use Prettier for formatting and can provide more information for us to ensure our program is consistent with their standards
+
+    - We will not have much access to their code, and not all API calls are available externally; however, Jasper can provide us with authorization for some requests (i.e., text-to-speech for audio files)
+
+<br>
```

Comment:
I think it could be because I haven't taken a look outside of the review mode, but I am not sure if the <br> does anything?